### PR TITLE
Allow meetings to be reopened by a Manager.

### PR DESCRIPTION
**Pull request by deiferni**
With this PR we add functionality to automatically reopen meetings. So far this was unsupported and had to be done manually via debug shell.

At the moment the action is only visible for a `Manager` as we want to test it out ourself before enabling it for customers.

A simple view will open which shows a summary of actions to be performed or reasons why they can't be performed at the moment.

I have translated only the action but not the view, as the view will only be used by us at the moment.
 
Screenshot 2021-04-27 at 10 45 46
Screenshot 2021-04-27 at 11 06 08

Jira: https://4teamwork.atlassian.net/browse/CA-1724

## Checklist

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

_Only applicable should be left and checked._

- Further improvements needed:
  - [x] Create follow-up stories and link them in the PR and Jira issue
- New translations
  - [x] All msg-strings are unicode

### CODE REVIEW

File: opengever/meeting/browser/meetings/reopen.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            else:
+                self.reopener.reopen_meeting()
+                msg = u'The meeting has been reopened.'
+                status.addStatusMessage(msg, type='info')
+
+            return self._redirect_to_meeting()
+
+    @button.buttonAndHandler(u'Cancel')
+    def handle_cancel(self, action):
+        self.prepare_reopener()
```
Comment (njohner) [inline opengever/meeting/browser/meetings/reopen.py:38]: ✂️

File: opengever/core/locales/fr/LC_MESSAGES/opengever.core.po [hunk 2]
```diff
@@ -259,6 +259,11 @@ msgstr "Requête"
 msgid "Proposal Template"
 msgstr "Modèle de requête"
 
+#: ./opengever/core/profiles/default/actions.xml
+#: ./opengever/core/upgrades/20210426170717_add_reopen_meeting_action/actions.xml
+msgid "Reopen meeting"
+msgstr ""
```
Comment (njohner) [inline opengever/core/locales/fr/LC_MESSAGES/opengever.core.po:265]: Rouvrir la séance

File: changes/CA-1724.other [hunk 3]
```diff
@@ -0,0 +1 @@
+Show action on meeting to reopen meeting for Manager. [deiferni]
```
Comment (njohner) [inline changes/CA-1724.other:1]: I think the major change is the introduction of the functionality, not actually showing the action...

**Review by njohner — APPROVED** [review]
LGTM

File: changes/CA-1724.other (hunk 3 above)
Comment (deiferni) [reply changes/CA-1724.other]: Yeah this is too action focused, changed it to be more generic. Thanks!

**Pull request merged**